content/applications/finance/payment_acquirers/asiapay.rst

```diff
@@ -0,0 +1,40 @@
+===========
+AsiaPay
+===========
+
+`AsiaPay <https://www.asiapay.com/>`_ is an online payments provider established in Hong Kong and
```

Comment:
```suggestion
`AsiaPay <https://www.asiapay.com/>`_ is an online payment acquirer established in Hong Kong and
```

Reply:
I'd keep `payment provider` since AsiaPay is indeed a "Payment Services Provider (PSP)" which offers a "payment gateway" = `payment acquirer` in Odoo. In 16.0, we will rename "acquirer" to "provider" everywhere to remove the confusion anyway.